src/detect-engine-tag.c

```diff
@@ -329,10 +329,11 @@ static void TagHandlePacketFlow(Flow *f, Packet *p)
                 case DETECT_TAG_METRIC_SECONDS:
                     /* last_ts handles this metric, but also a generic time based
                      * expiration to prevent dead sessions/hosts */
-                    if (iter->last_ts - iter->first_ts > iter->count) {
-                        SCLogDebug("flow tag expired: %u - %u = %u > %u",
-                            iter->last_ts, iter->first_ts,
-                            (iter->last_ts - iter->first_ts), iter->count);
+                    if (iter->last_ts.secs - iter->first_ts.secs > iter->count) {
```

Comment:
can we use SCTime_t funcs/macros to do this check?

Reply:
I did not find any, I will create one

Reply:
Oh I guess you mean `SCTIME_SECS` instead of directly accessing `.secs` field